source/fundamentals/typescript.txt

```diff
@@ -122,6 +121,45 @@ The type error raised by the preceding code snippet is as follows:
    ...
    Type '"Sylvilagus"' is not assignable to type 'Condition<"Canis" | "Felis">'.
 
+Creating Keys from Variables
+````````````````````````````
+
+If you need to query a collection or perform another operation using a
```

Comment:
Issue: [Style Guide discourages "using" on its own for ambiguity reasons](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#u)

```suggestion
If you need to query a collection or perform another operation with a
```